common/src/main/java/com/lightstep/tracer/shared/AbstractTracer.java

```diff
@@ -29,6 +29,7 @@
     // Maximum interval between reports
     private static final long DEFAULT_CLOCK_STATE_INTERVAL_MILLIS = 500;
     private static final int DEFAULT_REPORT_TIMEOUT_MILLIS = 10 * 1000;
+    private static final int DEFAULT_CLIENT_RESET_INTERVAL_MILLIS = 5 * 60 * 1000; // 5 min
```

Comment:
this is a good point, plus android clients probably don't need this at all because they can't realistically generate a high load. I'll add in an additional config which turns this off by default in the Android tracer.